src/coreComponents/mesh/generators/InternalWellGenerator.cpp

```diff
@@ -220,28 +223,45 @@ void InternalWellGenerator::constructPolylineNodeToSegmentMap()
     real64 const segmentLength = LvArray::tensorOps::l2Norm< 3 >( vSeg );
 
     // various checks and warnings on the segment and element length
-    GEOSX_THROW_IF( segmentLength < 1e-2,
-                    "Error in the topology of well " << getName() <<
-                    ": we detected a polyline segment measuring less than 1 cm",
+    GEOSX_THROW_IF( segmentLength < 1e-5,
+                    "Error in the topology of well '" << getName() <<
+                    "': we detected a polyline segment measuring less than 1 cm",
```

Comment:
Please update the error message accordingly (`1 cm` -> `segmentLength m`).

Reply:
I updated the error message.